feature-libs/customer-ticketing/assets/translations/en/customer-ticketing.ts

```diff
@@ -15,5 +17,12 @@ export const customerTicketing = {
     uploadFile: 'Upload File',
     fileSizeLimit: 'File Size Limit: {{count}}MB',
     maximumAttachment: 'Maximum of one attachment per message',
+    noTickets: `You don't have any request`,
+    requestTitle: 'All Requests',
+    sortSubtitle: 'Sort by',
+    // Mobile view
+    ticketIdFull: 'Ticket ID',
+    dateCreated: 'Date Created',
+    dateUpdated: 'Date Updated',
```

Comment:
Currently discussing with So.  We need to follow Desktop View.  IMO It should be "Created On" and "Changed On"
I am asking So to update Figma wires

Reply:
I have changed it to the way figma reflects now, using changed on and created on

Reply:
If changes were done, we no longer need these dateCreated nor dateUpdated. Please remove